### PR TITLE
[ECO-2005] Add WPR and info prompt

### PR DESCRIPTION
**Pull request by CRBl69**
# Description

This PR adds the display of WPR on pools page, as well as emojis when the
APR/WPR is ridiculous, with a popup to explain what they mean.

# Testing

See Pools page on vercel preview.

# Checklist

- [x] Did you check all checkboxes from the linked Linear task?

### CODE REVIEW

File: src/typescript/frontend/src/components/pages/pools/components/pools-table/components/table-header/index.tsx [hunk 1]
```diff
@@ -42,12 +47,26 @@ const TableHeader: React.FC<TableHeaderProps> = ({ item, isLast, onClick }) => {
...
+        <div className="relative">
+          <Image
+            src={info}
+            alt="info"
+            onTouchStart={() => setShowAPRInfoPrompt(!showAPRInfoPrompt)}
+            onMouseEnter={() => setShowAPRInfoPrompt(true)}
+            onMouseLeave={() => setShowAPRInfoPrompt(false)}
+          />
+          <Prompt
+            text="On extremly volatile markets, APR and WPR are impossible to predict. When such values are under -50% or above 1000%, they are represented by 😭 and 🤯 respectively."
```
Comment (xbtmatt) [inline src/typescript/frontend/src/components/pages/pools/components/pools-table/components/table-header/index.tsx:62]:
```suggestion
            text="On extremely volatile markets, APR and WPR are impossible to predict. When such values are under -50% or above 1000%, they are represented by 😭 and 🤯 respectively."
```

**Comment by CRBl69** [comment]
- Remove trailing zeros in `{D,W,A}PR`

**Review by xbtmatt — COMMENTED** [review]
Can you rebase/merge the latest changes, I'm unsure which changes are for this PR vs just not rebased yet

Also, I only see DPR, am I looking in the wrong place, or is it conditional or something

**Comment by CRBl69** [comment]
Updated. Note that the rest of the information is available on hover.

**Review by xbtmatt — APPROVED** [review]
I see the hover info now! I think you could probably extend the hover area to the whole column (aka the rest of the row, not just the text) but it's fine as is as well, don't want to block you on tiny things so up to you to add it

But yeah this is great, glad to have the radix tooltip stuff in here too, thanks for doing this, looks good! Much cleaner now 😄

**Pull request merged**